src/auth_manager/tests/AuthTest.cpp

```diff
@@ -0,0 +1,315 @@
+/**
+ * The following test cases are taken from OM.
+ * https://github.corp.ebay.com/SDS/om_cpp/blob/master/src/tests/unit/Middleware/AuthTest.cpp
```

Comment:
Please remove the internal reference to SDS or internal URL

Reply:
Sure, makes sense